src/exports.ts

```diff
@@ -1,12 +1,15 @@
+import fs from 'fs'
+import { join, resolve, dirname, extname } from 'path'
+import objectReplaceAll from 'objra'
```

Comment:
added `objra` ([objectReplaceAll](https://github.com/devjiwonchoi/objra)) to replace wildcards with entries

Reply:
Can we have some implmentation here as this is not very complex, and the package seems to be new published, kinda not want to depend on it